cmd/proxy/actions/app.go

```diff
@@ -137,6 +129,14 @@ func App(conf *config.Config) (*buffalo.App, error) {
 		app.Use(basicAuth(user, pass))
 	}
 
+	if !conf.FilterOff() {
```

Comment:
Wouldn't we validate a module even if it's on the `Exclude` list if the `ValidationMiddleware` is added **before** the `FilterMiddleware`?

Reply:
that;s right

Reply:
@marpio changed the order, please check if it makes sense for you now